content/develop/interact/search-and-query/advanced-concepts/vectors.md

```diff
@@ -9,369 +9,443 @@ categories:
+### Standard vector search
 
-**Invalid example** 
+Standard vector search has the following common syntax:
 
-`"(@title:Matrix)=>[KNN 10 @v $B] @year:[2020 2022]"`
+**Syntax**
 
-**Valid example** 
+```
+FT.SEARCH <index_name>
+  <primary_filter_query>=>[KNN <top_k> @<vector_field> $<vector_blob_param> $<vector_query_params> AS <distance_field>]
+  PARAMS <query_params_count> [$<vector_blob_param> <vector_blob> <query_param_name> <query_param_value> ...]
+  SORTBY <distance_field>
+  DIALECT 2
+```
+**Parameters**
 
-`"(@title:Matrix @year:[2020 2022])=>[KNN 10 @v $B]"`
+| Parameter         | Description                                                                                       |
+|:------------------|:--------------------------------------------------------------------------------------------------|
+| `index_name`  | Name of the index.  |
+| `primary_filter_query`  | [Filter]({{< baseurl >}}/develop/interact/search-and-query/advanced-concepts/vectors#filters) criteria. Use `*` when no filters are required.  |
+| `top_k` | Number of nearest neighbors to fetch from the index.  |
+| `vector_field`  | Name of the vector field in the index.  |
+| `vector_blob_param`  | The query vector as bytes and must be passed through the `PARAMS` section. The blob's byte size should match the vector field dimension and type.  |
+| `vector_query_params` (optional) | An optional section for marking one or more vector query parameters passed through the `PARAMS` section. Valid parameters should be provided as key-value pairs. See which [runtime query params]({{< baseurl >}}/develop/interact/search-and-query/advanced-concepts/vectors#runtime-query-params) are supported for each vector index type.  |
+| `distance_field` (optional) | The optional distance field name used in the response and/or for sorting. By default, the distance field name is "`__<vector_field>_score`" and it can be used for sorting without using `AS <distance_field>` in the query.  |
+| `vector_query_params_count` | The number of vector query params.  |
+| `vector_query_param_name` | The name of the vector query param.  |
+| `vector_query_param_value` | The value of the vector query param.  |
 
-The `<vector_similarity_query>` part inside the square brackets needs to be in the following format:
+**Example**
 
 ```
-KNN (<number> | $<number_attribute>) @<vector_field> $<blob_attribute> [<vector_query_param_name> <value>|$<value_attribute>] [...]] [ AS <dist_field_name> | $<dist_field_name_attribute>]
+FT.SEARCH my_index "(@title:Matrix @year:[2020 2022])=>[KNN 10 @vector $BLOB]" PARAMS 2 BLOB "\x12\xa9\xf5\x6c" DIALECT 2
 ```
 
-Every `*_attribute` parameter should refer to an attribute in the [`PARAMS`]({{< baseurl >}}/commands/ft.search/) section.
+**Use query attributes**
 
-* `<number> | $<number_attribute>` - Number of requested results ("K").
+Alternatively, as of v2.6, `<vector_query_params>` and `<distance_field>` name can be specified in runtime
```

Comment:
I want product @joeywhelan et al to comment on this because I believe we are going to be shifting how we discuss versions. This is the nomenclature present in the docs before the overhaul